test/controllers/email/EmailChangeSuccessControllerSpec.scala

```diff
@@ -146,4 +148,82 @@ class EmailChangeSuccessControllerSpec extends ControllerBaseSpec with MockConta
       }
     }
   }
+
+  "When the contactPrefMigration feature switch is turned off" when {
+
+    "calling the show action" when {
+
+      "a user is enrolled with a valid enrolment" when {
+
+        "the user has the email change success key in the session" when {
+
+          "a valid response is retrieved from the contact preference service" should {
+
+            "a digital preference is retrieved" should {
+
+              lazy val result = {
+                mockIndividualAuthorised()
+                mockConfig.features.contactPrefMigrationEnabled(false)
+                getMockContactPreference(vrn)(Right(ContactPreference("DIGITAL")))
+                mockGetEmailVerificationStatus(Some(true))
+                TestController.show(successfulChangeRequest)
+              }
+
+              "return 200" in {
+                status(result) shouldBe Status.OK
+              }
+
+              "audit the contact preference" in {
+                verifyExtendedAudit(ContactPreferenceAuditModel(vrn, "DIGITAL"))
+                reset(mockAuditingService)
+              }
+
+            }
+
+            "a paper preference is retrieved" should {
+
+              lazy val result = {
+                mockIndividualAuthorised()
+                mockConfig.features.contactPrefMigrationEnabled(false)
+                getMockContactPreference(vrn)(Right(ContactPreference("PAPER")))
+                mockGetEmailVerificationStatus(None)
+                TestController.show(successfulChangeRequest)
+              }
+
+              "return 200" in {
+                status(result) shouldBe Status.OK
+              }
+
+              "audit the contact preference" in {
+                verifyExtendedAudit(ContactPreferenceAuditModel(vrn, "PAPER"))
+                reset(mockAuditingService)
+              }
+
+            }
+
+            "an invalid response is retrieved from the contact preference service" should {
```

Comment:
Test description does not align with what the test is doing. The Contact Pref service is returning a successful response of "DIGITAL" in the mock below.